storagenode/storagenodedb/legacyinfo.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+const (
+	// LegacyInfoDBName represents the database name.
+	LegacyInfoDBName = "info"
+)
+
+// legacyInfoDB represents the database that contains the original legacy sqlite3 database.
+type legacyInfoDB struct {
+	SQLDB
+}
+
+func newLegacyInfoDB() *legacyInfoDB {
+	return &legacyInfoDB{}
+}
+
+// Configure sets the underlining SQLDB connection.
+func (db *legacyInfoDB) Configure(sqlDB SQLDB) {
+	db.SQLDB = sqlDB
+}
+
+// Rebind rebind parameters
+// These are implemented because the migrate.DB interface requires them.
+// Maybe in the future we should untangle those.
+func (db *legacyInfoDB) Rebind(s string) string { return s }
```

Comment:
If you add `Rebind` and `Schema` to `SQLDB` then you won't need to reimplement it for all of them. You should also be able to handle `Configure` such that there is no repetition.

Reply:
Alternatively there could be a separate type that implements these methods and all other databases would embed it.

Reply:
Done!